docs/sources/atlassian/confluence/confluence.yaml

```diff
@@ -0,0 +1,653 @@
+          $ref: "#/definitions/Links"
+        limit:
+          type: "integer"
+        results:
+          type: "array"
+          items:
+            $ref: "#/definitions/Task"
+        size:
+          type: "integer"
+        start:
+          type: "integer"
+  - pathTemplate: "/ex/confluence/{cloudId}/wiki/rest/api/content/search"
+    allowedQueryParams:
+      - "cql"
+      - "expand"
+      - "limit"
+      - "cursor"
+      - "includeArchivedSpaces"
+      - "start"
+    transforms:
+      - !<pseudonymize>
+        jsonPaths:
+          - "$.results[*]..accountId"
+          - "$.results[*]..email"
+          - "$.results[*]..attrs.id"
+        encoding: "URL_SAFE_TOKEN"
+      - !<redact>
+        jsonPaths:
+          - "$.results[*]..ari"
+          - "$.results[*]..displayName"
+          - "$.results[*]..publicName"
+          - "$.results[*]..profilePicture"
+          - "$.results[*]..body"
+          - "$.results[*]..base64EncodedAri"
+          - "$.results[*]..macroRenderedOutput"
+          - "$.results[*]..extensions"
+          - "$.results[*].._expandable"
+          - "$.results[*].._links"
+          - "$.results[*]..webui"
+          - "$.results[*]..self"
+          - "$.results[*]..download"
+          - "$._links.base"
+          - "$._links.context"
+          - "$._links.self"
+      - !<textDigest>
+        jsonPaths:
+          - "$.results[*]..title"
+          - "$.results[*].body.atlas_doc_format.value..text"
+          - "$.results[*]..message"
+    responseSchema:
+      type: "object"
+      properties:
+        _links:
+          $ref: "#/definitions/Links"
+        limit:
+          type: "integer"
+        results:
+          type: "array"
+          items:
+            $ref: "#/definitions/ContentItem"
+        size:
+          type: "integer"
+        start:
+          type: "integer"
+  - pathTemplate: "/ex/confluence/{cloudId}/wiki/rest/api/group"
+    allowedQueryParams:
+      - "limit"
+      - "start"
+    transforms:
+      - !<redact>
+        jsonPaths:
+          - "$..name"
+          - "$._links.base"
+          - "$._links.context"
+          - "$._links.self"
+    responseSchema:
+      type: "object"
+      properties:
+        _links:
+          $ref: "#/definitions/Links"
+        limit:
+          type: "integer"
+        results:
+          type: "array"
+          items:
+            $ref: "#/definitions/Group"
+        size:
+          type: "integer"
+        start:
+          type: "integer"
+  - pathTemplate: "/ex/confluence/{cloudId}/wiki/rest/api/group/{groupId}/membersByGroupId"
+    allowedQueryParams:
+      - "limit"
+      - "start"
+    transforms:
+      - !<pseudonymize>
+        jsonPaths:
+          - "$.results[*].accountId"
+          - "$.results[*].email"
+        encoding: "URL_SAFE_TOKEN"
+      - !<redact>
+        jsonPaths:
+          - "$.results[*].displayName"
+          - "$.results[*].profilePicture"
+          - "$.results[*].publicName"
```

Comment:
in theory, we shouldn't need these explicit redactions here - redundant with json-schema filter.